packages/sdk/src/utils/crossPlatformCrypto.ts

```diff
@@ -0,0 +1,10 @@
+import crypto from 'crypto'
+
+export function getSubtle(): crypto.webcrypto.SubtleCrypto {
+    const subtle = typeof window !== 'undefined' ? window?.crypto?.subtle : crypto.webcrypto.subtle
+    if (!subtle) {
+        const url = 'https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto'
+        throw new Error(`SubtleCrypto not supported. This feature is available only in secure contexts (HTTPS) & Node 16+. ${url}`)
+    }
+    return subtle
+}
```

Comment:
Is this part of ML-KEM pr?

Reply:
I see, it's a DRY refactor (subtle needed now in EncryptionUtil in addition to RSAKeyPair)